azext_iot/iothub/_help.py

```diff
@@ -309,3 +309,298 @@ def load_iothub_help():
         type: command
         short-summary: Upload a local file as a device to a pre-configured blob storage container.
     """
+
+    helps[
+        "iot hub message-endpoint"
+    ] = """
+        type: group
+        short-summary: Manage custom endpoints of an IoT hub.
+    """
+
+    helps[
+        "iot hub message-endpoint create"
+    ] = """
+        type: group
+        short-summary: Add an endpoint to an IoT Hub.
+    """
+
+    helps[
+        "iot hub message-endpoint create cosmosdb-collection"
```

Comment:
may need to add a check using ensure_iothub_sdk_min_version('2.3.0') to hide the help for older azure cli cores

Reply:
@digimaun just as an FYI - this throws a tiny wrench in the gears of hiding commands based on api-versions, we may also need to configure the help in a similar manner.

Reply:
Note - we discussed offline, this might be an issue just for our pipeline (assuming help commands don't show for commands that aren't loaded) and not actual customer usage - so we may be able to pop this in our local `linter_exclusions` to make our pipeline happy and be ok.